projects/kubernetes/kops/GIT_TAG

```diff
@@ -1 +1 @@
-v1.32.1
+v1.33.0-beta.1
```

Comment:
instead of using a beta version should we just add a patch doing the equivalent?